File: MsgClient.java
// CS 6421 - Simple Message Board Client in Java
// Yi Zhou
// Compile with: javac MsgClient
// Run with:     java MsgClient

import java.io.IOException;
import java.io.PrintWriter;
import java.net.Socket;
import java.net.UnknownHostException;

public class MsgClient {
    public static void main(String[] args) {
        String host = "finger92.koding.io";
        int portnum = 5555;

        try {
            Socket sock = new Socket(host, portnum);
            PrintWriter out = new PrintWriter(sock.getOutputStream(), true);
            out.println(args[args.length-2]);
            out.println(args[args.length-1]);
            
        } catch(Exception e) {
            System.out.println("Socket connection failed, Exception:"+e);
        }
       

    }
}

File: msgclient.py
# CS 6421 - Simple Message Board Client in Python
# Yi Zhou
# Run with:     python msgclient.py

import socket
import sys

host = "finger92.koding.io";
portnum = 5555;

sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM);   #get socket
sock.connect((host,portnum));   #get connetion
sock.send(sys.argv[1]+'\n'+sys.argv[2]); #send messages
sock.shutdown(socket.SHUT_WR);    #shutdown socket before close
sock.close();   #close socket

print("Sent message to server!")
